app/src/main/java/android/app/lotus/view/statistics/evaluation/Evaluation.kt

```diff
@@ -1,20 +1,301 @@
+                    fontSize = 20.sp,
+                    fontWeight = FontWeight.SemiBold
+                )
+            }
+        }
+// Add a Back button
+        val isBackButtonVisible = currentQuestion > 0
+
+                if (isBackButtonVisible) {
+                    Button(
+                        onClick = {
+                            currentQuestion = currentQuestion - 1
+                        },
+                        shape = RoundedCornerShape(25.dp),
+                        colors = ButtonDefaults.buttonColors(Color(148, 0, 124)),
+                        modifier = Modifier
+                            .padding(4.dp)
+                            .height(55.dp)
+                            .width(110.dp)
+
+                    ) {
+                        Text(
+                            text = "Back",
+                            fontSize = 20.sp,
+                            color = Color.White,
+                        )
+                    }
+                }
+            }
+    }
+
+@Composable
+fun OptionRadioButton(option: String,
+                      selectedOption: String,
+                      onOptionSelected: () -> Unit,
+                      borderColor: Color,
+                      shape: Shape,
+                      borderStrokeWidth: Dp,
+                      size: Dp
+) {
+    Row(
+        horizontalArrangement = Arrangement.Start, // Add spacing between options
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.padding(vertical = 4.dp)
+    ) {
+
+        Text(
+            text = option,
+            fontSize = 19.sp,
+            modifier = Modifier
+                .clickable { onOptionSelected() }
+                .padding(2.dp)
+        )
+        Spacer(modifier = Modifier.width(5.dp))
+        Box(modifier = Modifier
+            .size(size)
+            .border(border = BorderStroke(borderStrokeWidth, borderColor), shape = shape)
+            .background(Color.Transparent, shape = shape)
+            .clickable { onOptionSelected() })
+        if (option == selectedOption) {
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(borderColor, shape = shape)
+            )
+        }
+    }
+
+}
+@Composable
+fun YesNoOption(selectedOption: String, onOptionSelected: (String) -> Unit) {
+    Column(
+        modifier = Modifier.fillMaxWidth(),
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
 
+        OptionButton(
+            text = "Yes",
+            isSelected = selectedOption == "Yes",
+            onOptionSelected = { onOptionSelected("Yes") }
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+
+        OptionButton(
+            text = "No",
+            isSelected = selectedOption == "No",
+            onOptionSelected = { onOptionSelected("No") }
+        )
     }
-}
\ No newline at end of file
+}
+
+@Composable
```

Comment:
File is too long - move some of the buttons to their own files, if they can be reused in other views then put them under the general/ folder in view/

Reply:
if we gonna have two evaluation views one for employee and one for manager then i am gonna put them in general bc they are reusable